### ISSUE TITLE
Property overrides are not detected at all

### ISSUE THREAD

**Issue by tinder-tannerbennett** [Swiftify-Corp/Swiftify#186]
If I have this declaration:

```objc
@interface Person : NSObject

@property (nonatomic) BOOL fooBar;

@end
```

And in a subclass, I override the setter for `fooBar` like so:

```objc
@implementation Employee

- (void)setFooBar:(BOOL)fooBar {
    super.fooBar = fooBar;
    NSLog(@"Override");
}

@end
```

This is the code that gets generated:

```swift
class Employee: Person {
    func setFooBar(_ fooBar: Bool) {
        super.fooBar = fooBar
        print("Override")
    }
}
```

... but this is the expected result:

```swift
class Employee: Person {
    override var fooBar: Bool {
        get { super.fooBar }
        set {
            super.fooBar = fooBar
            print("Override")
        }
    }
}
```

**Comment by alex-swiftify** [comment]
Hey @tinder-tannerbennett, thanks for reporting!
We have already got similar reports - see
https://swiftify.atlassian.net/browse/SWC-1045#
https://github.com/Swiftify-Corp/Swiftify/issues/49
so we'll prioritize fixing this.

I'll keep you updated here.

**Comment by alex-swiftify** [comment]
@tinder-tannerbennett Good news - all issues fixed (will be available after the next update).

Input:
```
@interface Person : NSObject

@property (nonatomic) BOOL fooBar;

@end

@interface Employee : Person
@end

@implementation Employee

- (void)setFooBar:(BOOL)fooBar {
    super.fooBar = fooBar;
    NSLog(@"Override");
}

@end
```

Output:
```
class Person: NSObject {
    var fooBar = false
}

class Employee: Person {
    override var fooBar: Bool {
        get {
            super.fooBar
        }
        set(fooBar) {
            super.fooBar = fooBar
            print("Override")
        }
    }
}
```

**Important note**:
In order to get this translated properly, the converter should know that the `Employee` class derives from `Person` - thus I have included the following in the input code:
```
@interface Employee : Person
@end
```
Currently, this means that you should include both `Person` and `Employee` classes in the same conversion, i.e. check both classes in the Advanced Project Converter / Offline Converter.

I'll follow up with you once this is released, and your feedback is welcome, as always :)

**Comment by alex-swiftify** [comment]
Hi @tinder-tannerbennett ,

Just FYI - the changes to this ticket (#186), as well as #189, are now available in the Offline Converter version 5.5.2.
The app should normally update itself automatically, but you can also always install the latest version from our website.

Any feedback is welcome!
